templates/build.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html >
+<head>
+  <meta charset="UTF-8">
+  <title>Meilix Generator Email Notification</title>
+      <link rel="stylesheet" href="{{ url_for('static',filename='email.css') }}">
+</head>
+
+<body>
+  	<h2>Email will be sent with the downloadable ISO link once it is ready</h2>
+<main>
+	<div class="msg">1</div>
+	<section class="envelope-top"></section>
+	<section class="envelope-left"></section>
+	<section class="envelope-right"></section>
+</main>
+	<a class="tooltip" href="#">Check your email in 10 minutes!</a>
```

Comment:
i thought of adding feedback form here for user experience will remove in next pull